test/event_reporter_test.rb

```diff
@@ -16,35 +16,28 @@ def test_reporter_can_find_all_Johns
     assert_equal 63, result.count
   end
 
-  def test_queue_can_be_cleared
-    er = EventReporter.new('./data/full_event_attendees.csv')
-    list = List.new
-
-    er.find_attendees(:first_name, "Rachel")
-    refute er.list.empty?
-
-    list.clear_queue
-    assert_equal 0, list.queue_count
-  end
-
   def test_reporter_can_find_all_Marys_and_queue_them
     er = EventReporter.new('./data/full_event_attendees.csv')
-
     result = er.find_attendees(:first_name, "Mary")
 
```

Comment:
you could also assert that the first object in your list ACTUALLY has the first name of Mary.

```assert_equal "Mary", er.list.first.first_name```